requirements/main.in

```diff
@@ -41,7 +41,7 @@ paginate_sqlalchemy
 passlib>=1.6.4
 platformdirs>=4.2
 premailer
-psycopg[c]
+psycopg[binary]
```

Comment:
With this change, I don't think https://github.com/pypi/warehouse/blob/50fd637414dd90154e84d938a43d3515a1674550/.github/dependabot.yml#L25-L28 is needed anymore, is it?

Reply:
I think we still need to keep `psycopg` and `psycopg-binary` in sync?

Reply:
Ah yeah, I guess we still have the base package